compute/messaging/reference-content/nats-overview.mdx

```diff
@@ -0,0 +1,75 @@
+|:----------:|:----------------------------:|
+| Golang     | [nats.go](https://github.com/nats-io/nats.go), [godoc](http://godoc.org/github.com/nats-io/nats.go)               |
+| Java       | [nats.java](https://github.com/nats-io/nats.java), [javadoc](https://javadoc.io/doc/io.nats/jnats), [examples](https://github.com/nats-io/java-nats-examples) |
+| C#         | [nats.net](https://github.com/nats-io/nats.net), [doxygen](http://nats-io.github.io/nats.net/), [package](https://www.nuget.org/packages/NATS.Client/0.14.1)   |
+| JavaScript | [Node.js](https://github.com/nats-io/nats.js#readme), [deno](https://github.com/nats-io/nats.deno/blob/main/README.md), [WebSocket](https://github.com/nats-io/nats.ws#readme)     |
+| Python     | [nats.py](https://github.com/nats-io/nats.py), [doc](https://nats-io.github.io/nats.py/)                 |
+| Ruby       | [nats-pure.rb](https://github.com/nats-io/nats-pure.rb), [yard](https://www.rubydoc.info/gems/nats)           |
+| C          | [nats.c](https://github.com/nats-io/nats.c), [doc](http://nats-io.github.io/nats.c)                  |
+| Rust       | [nats.rs](https://github.com/nats-io/nats.rs), [doc](https://docs.rs/nats)            |
+| Elixir     | [nats.ex](https://github.com/nats-io/nats.ex), [doc](https://hex.pm/packages/gnat)             |
+| Zig        | [nats.zig](https://github.com/nats-io/nats.zig)                     |
+
+Further community NATS SDKs, e.g. SwiftyNats, Kotlin and Crystal, can be found [here](https://docs.nats.io/using-nats/developer).
+
+The following documents may help you get started using the above SDKs:
+- [Authentication with SDK](https://docs.nats.io/using-nats/developer/connecting/creds)
+- [Javascript: Using Jetstream](https://github.com/nats-io/nats.deno/blob/main/jetstream.md#jetstream-client)
+- [Python: Using Jetstream](https://github.com/nats-io/nats.py#jetstream)
+- [Go: Using Jetstream](https://github.com/nats-io/nats.go#jetstream-basic-usage)
+
+The main difference you will find between the standard NATS documentation and Scaleway Messaging is that we require you to provide a specific NATS URL and credentials to access the service . Please refer to our [developers documentation](https://developers.scaleway.com/en/products/messaging_and_queuing/api/v1alpha1/) for further information.
+
+## NATS Resources
+
+- [Official NATS Documentation: What is NATS?](https://docs.nats.io/nats-concepts/what-is-nats)
+- [Publish/Subscribe in pure NATS (no message retention)](https://docs.nats.io/nats-concepts/core-nats/pubsub/pubsub_walkthrough)
+- [Introduction to JetStream](https://docs.nats.io/nats-concepts/jetstream)
+
+Community NATS SDK can be found on https://docs.nats.io/using-nats/developer 
+Using SDKs
+Authentication with sdk: https://docs.nats.io/using-nats/developer/connecting/creds 
+Javascript: using jetstream https://github.com/nats-io/nats.deno/blob/main/jetstream.md#jetstream-client 
+Python: using jetstream ​​https://github.com/nats-io/nats.py#jetstream 
```

Comment:
Is it official / officially supported NATS doc?

Reply:
The author just go hired by synadia, the company developing nats, so it's getting official.
They may decide to move the site into nats.io later, but I think it will be easy enough to update the link.